### DISCUSSION TITLE
Using server for send-later activity - SOLVED workaround but in need of proper settings

### DISCUSSION THREAD

**Discussion by jakubstaszak-com** [Extended-Thunder/send-later#525] [Ideas]
Hi all,

due to my needs I run thunderbird on a mini server, that runs 24/7 anyway, maintaining multiple background tasks.

I also encounter a lot and lot of problems when Send Later was scheduled to send a message when my laptop was shut down.
A folder gets corrupted, a message has some weird body-issues, I get multiple error messages (sometimes as many as 50 or so) when opening thunderbird, messages are not sent but are placed in Sent folder, or they are sent but not in a folder etc.

I therefore want to run send later on a server -- as a sending service, as well as run it on my laptop as a scheduling service.

I've noticed that's not the default behaviour of Send Later, but I think it should be possible to set it that way.

First issue to solve is a profile

- Scheduled drafts are locked to a particular Thunderbird profile and will only be delivered by a Thunderbird running against the same profile that originally scheduled them. This means that if you use Thunderbird on multiple computers to schedule messages, you need to keep it running on all of those computers for the messages to be delivered. You can "transfer" a draft from one profile to another if need be by editing and rescheduling it.

Good thing my server profile is an exact copy of my laptop profile. Server is sending the messages just fine.

Another issue to solve is for laptop sendlater to not send messages, only work as a scheduling service.

Unfortunately, there's no trigger to do that, but I was able to set a sending frequency on a laptop to 1 year (in minutes, of course). If some random message happens to get double-sent once a year, it's not a big deal.

I think, it would be best for server-running Send later to be able intercept client-added messages from any profile and catch them for server's particular profile.

That way, not only does it require minimal configuration (tick a Server side checkbox in settings) but also prevents any double-sent messages, as well as allows client-PCs to sendlater their messages, that weren't intercepted by the server, on a default Send later behaviour.

**Comment by jikamens** [comment]
Two things:

1. You absolutely, positively should not have two Thunderbird profiles on different computers with the same Send Later UUID in them, unless all but one of them are configured not to send scheduled messages as described in [this section of the user guide](https://extended-thunder.github.io/send-later/#suppressing-scheduled-message-delivery).
2. The functionality you're looking for already exists. See [this section of the user guide](https://extended-thunder.github.io/send-later/#server-side-send-later).

**Comment by jakubstaszak-com** [comment]
Hi, thank you for your answer.
> 
>     1. You absolutely, positively should not have two Thunderbird profiles on different computers with the same Send Later UUID in them, unless all but one of them are configured not to send scheduled messages as described in [this section of the user guide](https://extended-thunder.github.io/send-later/#suppressing-scheduled-message-delivery).

Yes, it's part of workaround. I was able to set a sending frequency on a laptop to 1 year (in minutes, of course). If some random message happens to get double-sent once a year, it's not a big deal. Though setting it to 0 seems to better solution, I'm applying it right now.

>     2. The functionality you're looking for already exists. See [this section of the user guide](https://extended-thunder.github.io/send-later/#server-side-send-later).

Wow, OK, I must have missed that part looking for a solution and I'll give it a thorough read this week. Thank you!
